repos/spack_repo/builtin/packages/libmetatensor/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.cmake import CMakePackage, generator
+
+from spack.package import *
+
+
+class Libmetatensor(CMakePackage):
+    """Self-describing sparse tensor data format for atomistic machine learning and beyond."""
+
+    homepage = "https://docs.metatensor.org"
+    url = "https://github.com/metatensor/metatensor/releases/download/metatensor-core-v0.1.14/metatensor-core-cxx-0.1.14.tar.gz"
+    git = "https://github.com/metatensor/metatensor.git"
+
+    maintainers("HaoZeke", "luthaf", "rmeli")
+    license("BSD-3-Clause", checked_by="HaoZeke")
+
+    version("0.1.15", sha256="585cb8b67ea6b2d417690c30d16b9d097d280d09b10b11e8bc418c6ef78361e9")
+    version("0.1.14", sha256="dc6cdd9cf0113e2f012ecf68b81cc7cfc71bef3d2020b41574de8fa403dba646")
+    version("0.1.13", sha256="c735b1050357b2873e2e07ee1e263fc9d45faf07f5ea63b65e70869ca423adb5")
```

Comment:
See my review comment https://github.com/metatensor/metatensor/pull/925#pullrequestreview-3092002831 on a PR of libmetatensor-0.1.15 adding calls to `find_package_handle_standard_args` without
```cmake
include(FindPackageHandleStandardArgs)
```
Without it, this leads to 
```py
  CMake Error at /home/bkaindl/spack/opt/spack/linux-skylake/libmetatensor-0.1.15-irkj6wguphdshtnznf2fpp2q3c4arhwj/lib/cmake/metatensor/metatensor-config.cmake:108 (find_package_handle_standard_args):
    Unknown CMake command "find_package_handle_standard_args".
  Call Stack (most recent call first):
    CMakeLists.txt:28 (find_package)
```
when building metatensor-core with this PR.

I reported this to metatensor with https://github.com/metatensor/metatensor/issues/953

As I see no need to add 3 new versions when you should only need one version of the package, I suggest to remove the offending 0.1.15 (or You could also omit adding the older 0.1.13, btw.

```suggestion
    version("0.1.14", sha256="dc6cdd9cf0113e2f012ecf68b81cc7cfc71bef3d2020b41574de8fa403dba646")
```

Reply:
Thanks @bernhardkaindl. I also just tested building with Spack and can confirm I see the same issue with both `py-metatensor-core` and `libmetatensor-torch`.

@HaoZeke do you want me to have a look at these failures and provide fixes?

Reply:
Thanks for the quick fixes, I'll remove all but the latest